libs/ledger-live-common/src/families/solana/js-prepareTransaction.ts

```diff
@@ -105,15 +106,14 @@ const prepareTransaction = async (
     commandDescriptor,
   };
 
-  const preparedTx: Transaction = {
-    ...tx,
+  const patch: Partial<Transaction> = {
     model,
   };
 
   // LLM requires this field to be truthy to show fees
-  (preparedTx as any).networkInfo = true;
+  (patch as any).networkInfo = true;
```

Comment:
Why not adding a optional `networkInfo` field inside `Transaction` instead ?

Reply:
Because there's actually no need for that field in the transaction, except to make the UI happy.
In the end this is a hack, I don't know why we let it pass the review back then, but I feel it's better that it's obvious than to hide it deeper by artificially putting it in the Transaction type.